### ISSUE TITLE
Time sync is wrong

### ISSUE THREAD

**Issue by viktorbenei** [Parallels/vagrant-parallels#186]
Time sync only syncs the time but not the timezone. At the moment I'm in London, `date` prints `Fri May 29 17:24:13 BST 2015` on my Mac, when I run `date` in the vagrant managed Parallels Desktop VM I get the same time but with UTC timezone `Fri May 29 17:24:13 UTC 2015`. FYI BST is UTC+1 at the moment - and this timezone issue cases problems with communicating Amazon services.

The only solution so far is to turn time sync off. **edit** turning the time sync off in Parallels Desktop is only a very short term fix as PD10 probably does some CPU frequency optimization which messes up Ubuntu's own time sync and keeps adding an hour to the current time about every minute!

So the actual solution what worked is to set the timezone manually inside the VM (ubuntu: `sudo timedatectl set-timezone Europe/London`) and turn Parallels time sync **on** for the VM.

## Tech info

- OS X: 10.10.3
- Vagrant 1.7.2
- vagrant-parallels (1.3.12)
- Parallels Desktop: 10.2.0 (28956), 10.2.1 (29006) - tested with both
- Ubuntu 14.04 VM

**Comment by legal90** [comment]
@viktorbenei Thank you for report! 
I've reproduced this issue and forwarded it to the responsible people in Parallels Desktop. I will let you know about any updates on this.

**Comment by viktorbenei** [comment]
Thanks!

**Comment by fredericgaus** [comment]
Any news on this topic? Setting the timezone is not working for me because of daylight saving adjustments :-(

**Comment by viktorbenei** [comment]
@fredericgaus as a temporary solution you could use `sudo timedatectl set-timezone Europe/London`, this should take care of daylight saving AFAIK

**Comment by fredericgaus** [comment]
@viktorbenei thanks for your answer. On Debian `timedatectl` is not available. This is why I used `dpkg-reconfigure tzdata`as well as `ln -sf /usr/share/zoneinfo/Europe/Berlin /etc/localtime` but both don't do the trick :(.

For some reasons the synchronized time also seems to be wrong or not UTC as expected by Debian as I get the following:

```
Current default time zone: 'Europe/Berlin'
Local time is now:      Fri Jun 19 10:56:16 CEST 2015.
Universal Time is now:  Fri Jun 19 08:56:16 UTC 2015.
```

But the  current UTC time is 7:56:16. So the synchronized UTC time is already wrong. 

In my Ubuntu, when I do `sudo timedatectl set-timezone Europe/Berlin` the clock is set correctly but is reseted to the wrong value as it was before about a minute later. Any ideas?

**Comment by viktorbenei** [comment]
@fredericgaus what worked for me was to **enable** the Parallels time sync feature for the Ubuntu VM, then SSH into the VM and call `sudo timedatectl set-timezone Europe/London`. With the versions I listed in the Tech Info section it worked for me, `timedatectl` sets the time zone and after that Parallel's built in time sync works correctly as it does not modify the timezone, only the time is synced from the host machine (at least in my case).

**Comment by fredericgaus** [comment]
@viktorbenei thanks you reply. Sadly the Parallels time sync in my case do adjust the timezone to EET which is wrong. (My host runs on CEST).

So this is not working for me :(

**Comment by legal90** [comment]
@fredericgaus Unfortunately, the only tip I can get you now is to disable Time Sync feature and synchronize time on the VM side with `ntpd`.
How to disable Time Sync (works on Parallels Desktop >= 10.1.2):
```ruby
config.vm.provider "parallels" do |prl|
  prl.customize ["set", :id, "--time-sync", "off"]
end
```

**Comment by fredericgaus** [comment]
@legal90 thanks for the tip. As explained in the original issue report this does not work either. If I do, every minute the hours increase too. Doing an ntpd update every minute is not reliable enough then. Doing it every second is too much. I fear the only option I got currently is to get along with the much slower VirtualBox :-1:

**Comment by smurfy** [comment]
I have the same problem, (CEST timezone) Now switched my Mac "Closest City" to ```UTC - United Kingdom``` showing the Time Zone in the dialog as ```GMT``` and in terminal as ```UTC```.

Parallels seams to sync the Time and Timezone correctly and i now have UTC zone and the same time as my mac. just need to remember that i need to add 2 hours to my time :)

**Comment by viktorbenei** [comment]
FYI at the moment we use this solution :

We don't install the Parallels Tools, but rather we do a manual time sync after rolling back to a snapshot state, sending a `sudo date` through ssh. 

You can find the code here, if that helps: https://github.com/bitrise-io/bitrise-machine/blob/dc80e95c61df322c27b8738287b69d69eb3cca89/cli/setup.go#L67

The point is, to not to mess with the timezone settings inside the VM you have to pass the time with the timezone information included.

**Comment by legal90** [comment]
Fixed in [v1.5.0](https://github.com/Parallels/vagrant-parallels/releases/tag/v1.5.0)
For Parallels Desktop 10 the timesync feature is enabled by default:
```
--time-sync off
```

For Parallels Desktop 11+ it's enabled, but timezones are not affected:
```
--time-sync on
--disable-timezone-sync on
```
